apertium-init.py

```diff
@@ -242,6 +236,12 @@ def main(cli_args):  # type: (List[str]) -> None
     else:
         parser.error('Invalid language module name: %s' % args.name)
 
+    if os.path.exists(args.destination):
+        sys.stderr.write('Directory {} already exists, quitting.\n'.format(args.destination))
```

Comment:
Is there a reason this was moved?

Reply:
where it was before, if an invalid name (eng-cat-spa) was supplied, it would create the directory before giving an error and not delete it. Now it will only create the directory if it's actually creating a module.

Reply:
Ah, great fix!